recipes/multiqc/meta.yaml

```diff
@@ -76,8 +79,6 @@ extra:
     # click requires a unicode locale when used with Python 3
     # extended-base generates en_US.UTF-8 locale and sets LC_ALL, LANG properly
     extended-base: true
```

Comment:
I'm not convinced that this is really needed now. Does anyone else have recent experience of bioconda packages using Click?

It would be nice to remove this and get smaller bioconda images. I also wonder if using a different image base could fix the Docker issues happening recently in GitPod: https://github.com/MultiQC/MultiQC/issues/2283

Reply:
not sure either - should we just try ?

Reply:
I'm using rich-click and usually that's enough (dont need tqdm, pydantic, typeguard)